hack/build-and-push.sh

```diff
@@ -48,7 +48,7 @@ fi
 
 APPSTUDIO_UTILS_IMG="quay.io/$MY_QUAY_USER/${TEST_REPO_NAME:-appstudio-utils}:${TEST_REPO_NAME:+build-definitions-utils-}$BUILD_TAG"
 APPSTUDIO_TASKS_REPO=quay.io/$MY_QUAY_USER/${TEST_REPO_NAME:-appstudio-tasks}
-PIPELINE_BUNDLE_IMG=quay.io/$MY_QUAY_USER/${TEST_REPO_NAME:-build-templates-bundle}:${TEST_REPO_NAME:+base-}$BUILD_TAG
+PIPELINE_BUNDLE_IMG=quay.io/$MY_QUAY_USER/${TEST_REPO_NAME:-build-templates-bundle}:${TEST_REPO_NAME:+build-}$BUILD_TAG
```

Comment:
@Michkov I just checked that we actually have one missing piece (that I missed!) in order to test HACBS pipeline.

We have at least 2 options:
* go with this change (e2e-test already takes care of updating the bundle configmap to hacbs one, so this works) OR
* update [line 110](https://github.com/redhat-appstudio/build-definitions/blob/430450dcd2dbb3846be9c65c8eed515291e44a6c/hack/build-and-push.sh#L110) to install `$HACBS_BUNDLE_IMG` instead of `$PIPELINE_BUNDLE_IMG` by default

wdyt?

Reply:
could be like this